include/bbp/sonata/population.h

```diff
@@ -50,6 +50,9 @@ class SONATA_API Selection
     const Ranges ranges_;
 };
 
+bool SONATA_API operator==(const Selection&, const Selection&);
+bool SONATA_API operator!=(const Selection&, const Selection&);
```

Comment:
Can we also have `operator bool`?

Reply:
It is in the Python bindings, but it feels weird to me to override this operator in C++. One should use the `empty()` method like for STL containers: `if (!my_vector.empty())`.

Reply:
we already have `empty()` like in std::vector

Reply:
Or what @tristan0x said.

Reply:
Maybe I just like my C++ code to feel more Pythonic…

Reply:
I try and follow the principle of least astonishment, and in the C++ community, my read is that containers have an `empty()` and having a bool operator would be surprising.

Reply:
…least astonishment…C++… that's a good one. Anyways, fine with me, I don't have strong feelings here.

Reply:
Thank you for your comment anyway @matz-e